manager/src/main/java/io/gingersnapproject/util/UniItem.java

```diff
@@ -0,0 +1,54 @@
+package io.gingersnapproject.util;
```

Comment:
Nitpick, but is `.mutiny` a more meaningful package name?

Reply:
I was planning on putting various utility things here, but sure I can change it.